prospero-cli/src/test/java/org/wildfly/prospero/cli/commands/ChannelCommandTest.java

```diff
@@ -49,6 +51,7 @@ public class ChannelCommandTest extends AbstractConsoleTest {
     private static final MavenCoordinate GA = new MavenCoordinate("g", "a", null);
     private static final MavenCoordinate GAV = new MavenCoordinate("g", "a", "v");
     private static final String URL = "file:/a:b";
+    private final URL testChannelURL = getClass().getClassLoader().getResource("test-channel.yaml");
```

Comment:
@spyrkob I am using static yaml file as it is reusable and seems easier. WDT?

Reply:
You can simply use `final Path path = MetadataTestUtils.prepareChannel("manifest.yaml");` instead. I'd rather avoid adding more channel definitions that need to be maintained manually.

Also, as far as I can tell, this field is not used at all.